pkg/flowcli/config/json/account.go

```diff
@@ -39,45 +40,75 @@ func transformAddress(address string) flow.Address {
 	return flow.HexToAddress(address)
 }
 
+// transformSimpleToConfig transforms simple internal account to config account
+func transformSimpleToConfig(accountName string, a simpleAccount) config.Account {
+	pkey, _ := crypto.DecodePrivateKeyHex(
+		crypto.ECDSA_P256,
+		strings.ReplaceAll(a.Key, "0x", ""),
+	)
+
+	return config.Account{
+		Name:    accountName,
+		Address: transformAddress(a.Address),
+		Key: config.AccountKey{
+			Type:       config.KeyTypeHex,
+			Index:      0,
+			SigAlgo:    crypto.ECDSA_P256,
+			HashAlgo:   crypto.SHA3_256,
+			ResourceID: "",
+			PrivateKey: pkey,
+		},
+	}
+}
+
+// transformAdvancedToConfig transforms advanced internal account to config account
+func transformAdvancedToConfig(accountName string, a advanceAccount) config.Account {
+	sigAlgo := crypto.StringToSignatureAlgorithm(a.Key.SigAlgo)
+	var pKey crypto.PrivateKey
+	resourceID := a.Key.ResourceID
+
+	// todo check both pkey and resource id if present and return error if both
+
+	if a.Key.PrivateKey != "" {
+		pKey, _ = crypto.DecodePrivateKeyHex(
+			sigAlgo,
+			strings.ReplaceAll(a.Key.PrivateKey, "0x", ""),
+		)
+	}
+
+	// pre v0.22 support
+	if a.Key.Context["privateKey"] != "" {
+		pKey, _ = crypto.DecodePrivateKeyHex(
+			sigAlgo,
+			strings.ReplaceAll(a.Key.Context["privateKey"], "0x", ""),
+		)
+	}
+
+	return config.Account{
+		Name:    accountName,
+		Address: transformAddress(a.Address),
+		Key: config.AccountKey{
+			Type:       a.Key.Type,
+			Index:      a.Key.Index,
+			SigAlgo:    sigAlgo,
+			HashAlgo:   crypto.StringToHashAlgorithm(a.Key.HashAlgo),
+			ResourceID: resourceID,
+			PrivateKey: pKey,
+		},
+	}
+}
+
+// todo add error in return and do validation
```

Comment:
```suggestion
// TODO: Add error in return and do validation
```

I think some editors only pick up todos in this format